nsds_lab_to_nwb/metadata/resources/list_of_stimuli.yaml

```diff
@@ -41,8 +41,22 @@ wn2:
     parameter_path: ''
```

Comment:
@vngutierrez This one is missing as well. Is there a Tone .mat file you can find?

Reply:
There is no Tone.mat file that I can find, only .wav files.